### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: packages/extension/src/background/alarms.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import browser from "webextension-polyfill"
+
+const transactionTrackerHistoryAlarm = "core:transactionTracker:history"
+export async function refreshTransactionTrackerHistoryAlarm() {
```
Comment (h0ngcha0) [inline packages/extension/src/background/alarms.ts:4]: [Nit] This function doesn't seem to be used.

File: packages/extension/src/background/alarms.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+export function setTransactionTrackerHistoryAlarm() {
+  browser.alarms.clear(transactionTrackerHistoryAlarm)
+  browser.alarms.create(transactionTrackerHistoryAlarm, {
+    periodInMinutes: 5, // fetch history transactions every 5 minutes from voyager
+  })
+}
+
+const transactionTrackerUpdateAlarm = "core:transactionTracker:update"
+export async function refreshTransactionTrackerUpdate() {
```
Comment (h0ngcha0) [inline packages/extension/src/background/alarms.ts:16]: [Nit] `refreshTransactionTrackerUpdate` -> `refreshTransactionTrackerUpdateAlarm`

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: packages/extension/src/background/alarms.ts (hunk 1 above)
Comment (polarker) [reply packages/extension/src/background/alarms.ts:4]: Maybe keep it just for consistency

File: packages/extension/src/shared/transactions.ts [hunk 3]
```diff
@@ -5,17 +5,19 @@ import { ReviewTransactionResult } from "./actionQueue/types"
 import { WalletAccount } from "./wallet.model"
 import { AlephiumExplorerTransaction } from "./explorer/type"
 
-export type Status = 'NOT_RECEIVED' | 'RECEIVED' | 'PENDING' | 'ACCEPTED_ON_L2' | 'ACCEPTED_ON_CHAIN' | 'REJECTED';
+export type Status = 'NOT_RECEIVED' | 'RECEIVED' | 'PENDING' | 'ACCEPTED_ON_MEMPOOL' | 'ACCEPTED_ON_L2' | 'ACCEPTED_ON_CHAIN' | 'REJECTED';
```
Comment (Lbqds) [inline packages/extension/src/shared/transactions.ts:8]: Do we need to remove the `ACCEPTED_ON_L2`?

**Review by Lbqds — APPROVED** [review]
LGTM

File: packages/extension/src/shared/transactions.ts (hunk 3 above)
Comment (polarker) [reply packages/extension/src/shared/transactions.ts:8]: Yes, but it will break a lot of other code. So we keep them for now.

**Pull request merged**